src/components/snapshot/overview/SnapshotExport.test.tsx

```diff
@@ -48,8 +48,11 @@ describe('SnapshotExport', () => {
       </Router>,
     );
     cy.get('[data-cy="export-snapshot-button"]').should('contain.text', 'Export snapshot');
+    cy.get('[data-cy="tdr-sync-permissions-checkbox"] input')
```

Comment:
Note that we're using a MUI Checkbox and need to drill down to the underlying `input` before performing our assertions.